src/components/Calculator/Calculator.jsx

```diff
@@ -21,27 +21,30 @@ const Calculator = () => {
 
   // RENDER
   return (
-    <div id="calculator-container">
-      <span className="dispaly">{display}</span>
-      <Button className="btn" label="AC" onClick={() => handleClick('AC')} />
-      <Button className="btn" label="+/-" onClick={() => handleClick('+/-')} />
-      <Button className="btn" label="%" onClick={() => handleClick('%')} />
-      <Button className="btn btn-right" label="÷" onClick={() => handleClick('÷')} />
-      <Button className="btn" label="7" onClick={() => handleClick('7')} />
-      <Button className="btn" label="8" onClick={() => handleClick('8')} />
-      <Button className="btn" label="9" onClick={() => handleClick('9')} />
-      <Button className="btn btn-right" label="x" onClick={() => handleClick('x')} />
-      <Button className="btn" label="4" onClick={() => handleClick('4')} />
-      <Button className="btn" label="5" onClick={() => handleClick('5')} />
-      <Button className="btn" label="6" onClick={() => handleClick('6')} />
-      <Button className="btn btn-right" label="-" onClick={() => handleClick('-')} />
-      <Button className="btn" label="1" onClick={() => handleClick('1')} />
-      <Button className="btn" label="2" onClick={() => handleClick('2')} />
-      <Button className="btn" label="3" onClick={() => handleClick('3')} />
-      <Button className="btn btn-right" label="+" onClick={() => handleClick('+')} />
-      <Button className="btn zero" label="0" onClick={() => handleClick('0')} />
-      <Button className="btn" label="." onClick={() => handleClick('.')} />
-      <Button className="btn btn-right" label="=" onClick={() => handleClick('=')} />
+    <div className="calculator">
+      <h2>Let&apos;s do some math!</h2>
```

Comment:
> - [ ] Please make a small adjustment to the text so that it aligns with the navbar, as shown in the design layout.👍
> 
> ![ox1](https://github.com/HossainAraf/math-magician/assets/40009354/37c86fd2-1491-4819-aa41-bf8bb3adb91b)